Started by user [8mha:////4Gm8z8svCaXKqFzr1Z5zntsQFmxEgC1XfjqScehTkFs8AAAAlx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzWEgZu/dLi1CL9xJTczDwAj6GcLcAAAAA=[0mnazar
Running as SYSTEM
Building remotely on [8mha:////4Gb5jqnYN5atl4H6TvVp5YbnrSRqULYVnZcIQhut4W5hAAAAnx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzOEgZh/eT83ILSktQi/TBfI93inMSyVAA+LWVRyAAAAA==[0mVM2-slave (linux_vm2_slave) in workspace /home/user1/workspace/ansible-docker
[ansible-docker] $ /bin/sh -xe /tmp/jenkins6171042636383977770.sh
+ echo 172.31.38.161 ip-172-31-38-161.us-east-2.compute.internal 172.31.32.48 ip-172-31-32-48.us-east-2.compute.internal
172.31.38.161 ip-172-31-38-161.us-east-2.compute.internal 172.31.32.48 ip-172-31-32-48.us-east-2.compute.internal
+ sudo chmod 666 /etc/ansible/hosts
+ sudo sed -i '/ansibleslave\]/d' /etc/ansible/hosts
+ sudo sed -i /172.31.32.48/d /etc/ansible/hosts
+ sudo echo '[ansibleslave]'
+ sudo echo 172.31.32.48
[ansible-docker] $ /bin/sh -xe /tmp/jenkins2439219191641412629.sh
+ sudo touch /etc/ansible/playbook-docker-uninstall.yml
+ sudo chmod 755 /etc/ansible/playbook-docker-uninstall.yml
+ sudo tee /etc/ansible/playbook-docker-uninstall.yml
[ansible-docker] $ sshpass ******** ansible-playbook /etc/ansible/playbook-docker-uninstall.yml -f 5 -u user1 -k

PLAY [ansibleslave] ************************************************************

TASK [Gathering Facts] *********************************************************
[WARNING]: Platform linux on host 172.31.32.48 is using the discovered Python
interpreter at /usr/bin/python, but future installation of another Python
interpreter could change this. See https://docs.ansible.com/ansible/2.9/referen
ce_appendices/interpreter_discovery.html for more information.
ok: [172.31.32.48]

TASK [Stop docker] *************************************************************
changed: [172.31.32.48]

TASK [Uninstall docker] ********************************************************
changed: [172.31.32.48]

TASK [Install docker] **********************************************************
changed: [172.31.32.48]

TASK [Start docker] ************************************************************
changed: [172.31.32.48]

PLAY RECAP *********************************************************************
172.31.32.48               : ok=5    changed=4    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

[ansible-docker] $ /bin/sh -xe /tmp/jenkins4690021709537326557.sh
+ build_number=45
+ sudo touch /etc/ansible/playbook-docker.yml
+ sudo chmod 755 /etc/ansible/playbook-docker.yml
+ sudo tee /etc/ansible/playbook-docker.yml
+ echo project45
project45
+ echo project45
project45
[ansible-docker] $ sshpass ******** ansible-playbook /etc/ansible/playbook-docker.yml -f 5 -u user1 -k

PLAY [ansibleslave] ************************************************************

TASK [Gathering Facts] *********************************************************
[WARNING]: Platform linux on host 172.31.32.48 is using the discovered Python
interpreter at /usr/bin/python, but future installation of another Python
interpreter could change this. See https://docs.ansible.com/ansible/2.9/referen
ce_appendices/interpreter_discovery.html for more information.
ok: [172.31.32.48]

TASK [Creates directory] *******************************************************
changed: [172.31.32.48]

TASK [setting up gitrepo] ******************************************************
changed: [172.31.32.48]

TASK [Get the project from github] *********************************************
changed: [172.31.32.48]

PLAY RECAP *********************************************************************
172.31.32.48               : ok=4    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

[ansible-docker] $ /bin/sh -xe /tmp/jenkins5273360648645772748.sh
++ ls /home/user1/project/
ls: cannot access /home/user1/project/: No such file or directory
+ projectname=
Build step 'Execute shell' marked build as failure
Finished: FAILURE
